vars/govuk.groovy

```diff
@@ -160,7 +160,7 @@ def nonDockerBuildTasks(options, jobName, repoName) {
 
   if (hasLint()) {
     stage("Lint Ruby") {
-      rubyLinter("app lib spec test")
+      rubyLinter("app lib spec test", options.get('rubyLintDiff', true))
```

Comment:
I think we should keep it, if it's in the same line as you did some code.
This will make the repo closer to the current style and closer to consistency

Reply:
@leenagupte I've updated the `README.md` now :)